packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,52 +1,138 @@
+
 ## Background
 
-_Description and use cases of this component_
+An InfoButton provides a way for users to get more information about a particular UI element. It is a button that displays an icon and when clicked a Popover displays the additional information.
+
+Because the Popover isn't always visible, it should not contain information that people must know in order to complete the field.
 
 ## Prior Art
 
 _Include background research done for this component_
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+#### Open UI
+
+| Name                                                | library                                                                               | Notes |
+| --------------------------------------------------- | ------------------------------------------------------------------------------------- | ----- |
+| IconButtonTooltip                                   | [Carbon Design System](https://www.carbondesignsystem.com/components/tooltip/usage)   |       |
+| Tooltip with card appearance                        | [EverGreen](https://evergreen.segment.com/components/tooltip)                         |       |
+| Input and guidance to add info button               | [Salesforce](https://www.lightningdesignsystem.com/components/input/)                 |       |
+| TextField with example on how to add an info button | [v8](https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield)          |       |
+| ContextualHelp                                      | [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html) |       |
+
+#### Comparison with v0 and v8
+
+- v0 does not have an InfoButton component.
+- v8 does not have an InfoButton component, but does have a TextField component that has an example of how to add an info button to a TextField, see Open UI section above for more details.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+<InfoButton content="This is some additional information." />
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+Visual (only the button has a variant)
+
+- `transparent` (default)
+- `subtle`
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+#### Props
+
+```ts
+import type { Button } from '@fluentui/react-button';
+import type { PopoverProps, PopoverSurface } from '@fluentui/react-popover';
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type InfoButtonSlots = {
+  root: NonNullable<Slot<PopoverProps>>;
+
+  /**
+   * The button that triggers the Popover.
+   */
+  trigger: NonNullable<Slot<typeof Button>>;
```

Comment:
Let's avoid calling this `trigger` to avoid confusion with the triggers pattern.